src/test/integration/utils/mocha.js

```diff
@@ -61,7 +70,10 @@ async function createIframe() {
   container.innerHTML = "";
   container.appendChild(iframe);
 
-  await waitForTime(2000);
+  await waitForLoad(iframe);
```

Comment:
I think we could have a race condition here, if onload is fired before we hit this line (which could be the case since onload is fired when we add the iframe to the document).
I don't know if it's very likely though, but I'd feel safer if we could declare declare the onload handler before adding the iframe to the container.